packages/wallet/backend/src/open-api/config.ts

```diff
@@ -0,0 +1,41 @@
+import swaggerJsdoc from 'swagger-jsdoc'
+import { AccountPaths, AccountSchema } from '@/account/account.docs'
+const options = {
+  definition: {
+    openapi: '3.1.0',
+    info: {
+      title: 'Rafiki Wallet Express API with Swagger',
+      version: '0.1.0',
+      description:
+        'This is a simple CRUD API application made with Express and documented with Swagger',
+      license: {
+        name: 'MIT',
+        url: 'https://spdx.org/licenses/MIT.html'
+      },
+      contact: {
```

Comment:
what is this?

Reply:
No We should fill it with valid contacts